### ISSUE TITLE
[Bug]: Out-of-the-box installation doesn't read the default dialect config

### ISSUE THREAD

**Issue by uuouter** [grammarly/grammarly-for-developers#218]
### What package(s) are you seeing the problem in?

@grammarly/editor-sdk

### SDK version

2.0.3

### Browser(s)

Safari

### Operating system(s)

macOS

### Steps to reproduce

1. Install Grammarly Editor SDK with NPM
2. Init a textarea using either the web component or through javascript

### What happened?

Grammarly is warning me about using a non-British variant while the default setting is American.

![image](https://user-images.githubusercontent.com/18463843/195417907-ed450e86-296a-4d38-bf9f-bc64d84ef941.png)

Reselecting the dialect in the menu does fix this issue.

### What did you expect to happen?

No warning

**Comment by wachunga** [comment]
Thanks for the report, @uuouter. The [default setting for dialect is "auto-text"](https://developer.grammarly.com/docs/api/editor-sdk/editorconfig#documentdialect), which may be detecting British English, but unfortunately the Grammarly menu select doesn't reflect that.

Failing that, it may be because of the accept-language header that's coming from your browser. 

To help us debug, could you do the following?
1. Paste the output of `navigator.languages` from your browser console
2. Paste the text you're using to help reproduce the issue

**Comment by uuouter** [comment]
@wachunga Thanks for the quick response! Absolutely.

navigator.languages
["en-GB"] (1) = $1

Seems there is the issue, so the menu doesn't reflect it.

Text:

```
That said, with apps such as these, content is king. Well, and the rewards.

Can they keep it interesting?

Your friendly Dutch neighbor,

– W
```

**Comment by wachunga** [comment]
@uuouter We've narrowed down the issue and working on a fix (probably early next week). In the meantime, you can manually set American English as the default dialect using [EditorConfig](https://developer.grammarly.com/docs/api/editor-sdk/editorconfig):

```html
<grammarly-editor-plugin
  config.documentDialect="american"
>
  <textarea></textarea>
</grammarly-editor-plugin>
```

**Comment by wachunga** [comment]
Hi @uuouter, this has now been fixed in [2.1.0](https://developer.grammarly.com/docs/changelog/#_2-1-0-2022-11-02). The Grammarly button menu should now correctly reflect the actual dialect that Grammarly is using to check your text.

Thanks for reporting the bug!
